console/console-init/ui/src/graphql-module/queries/Address.ts

```diff
@@ -64,10 +64,18 @@ const ALL_ADDRESS_FOR_ADDRESS_SPACE_FILTER = (
     filter += " AND ";
   }
 
+  //filter status
   if (statusValue) {
-    filter += generateFilterPattern("status.phase", [
-      { value: statusValue, isExact: true }
-    ]);
+    if (statusValue !== "Pending") {
+      filter += generateFilterPattern("status.phase", [
+        { value: statusValue, isExact: true }
+      ]);
+    } else {
+      filter += generateFilterPattern("status.phase", [
+        { value: statusValue, isExact: true },
+        { value: "", isExact: true }
```

Comment:
is there any reason for adding second object with black value ?

Reply:
Yes there is. When controller didn't pick the address or address-space for creation, the server assgin status.phase as empty string and in the UI we are displaying the value as Pending.